README.md

```diff
@@ -62,6 +62,10 @@ function NewComponent(props) {
 }
 ```
 
+Some things in the Design System, such as the universal focus styling and the box-model the DS uses, are namespaced to the `.nypl-ds` class. To include those in your app, add `.nypl-ds` to whichever wrapper tag makes sense in your application (e.g., `<html>`, `<body>`, or `<div class="app">`).
+
+Please note that the NYPL Header and Footer should be _outside_ of the `.nypl-ds` wrapper class.
```

Comment:
This is fine but note that above it says it can go in `<body>` but the header and footer _need_ to go in the body element. I don't think this is a big deal yet and we also don't know how other apps will use it.